recipes/lmdb/all/CMakeLists.txt

```
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# LMDB library:
+
+add_library(lmdb
+    source_subfolder/lmdb.h
+    source_subfolder/mdb.c
+    source_subfolder/midl.c
+    source_subfolder/midl.h
+)
+
+set_target_properties(lmdb PROPERTIES
+    PUBLIC_HEADER source_subfolder/lmdb.h
+    WINDOWS_EXPORT_ALL_SYMBOLS ON
+)
+
+if(NOT WIN32)
+    target_link_libraries(lmdb PUBLIC pthread)
+endif()
```

```suggestion
if(NOT WIN32)
    target_link_libraries(lmdb PUBLIC pthread)
endif()
```

Prefer [FindThreads](https://cmake.org/cmake/help/v3.1/module/FindThreads.html) instead. It's more generic

---

All right.